lib/Locale/Country/Extra.pm

```diff
@@ -33,15 +32,11 @@ sub country_from_code {
 sub code_from_country {
     my ($self, $country) = @_;
 
-    my %code_countries = reverse %{ $self->_country_codes };
+    my $lcm  = Locale::Country::Multilingual->new();
+    my $code = $lcm->country2code($country);
```

Comment:
These two lines seem similar enough to the `localized_code2country()` below that it might be worth refactoring a bit.

Reply:
change to one line ?
Locale::Country::Multilingual->new()->country2code($country);